src/hooks/useDocumentTitle.ts

```diff
@@ -0,0 +1,7 @@
+import { useEffect } from 'react';
+
+export default function useDocumentTitle(title: string): void {
+  useEffect(() => {
+    document.title = title;
+  }, [title]);
+}
```

Comment:
nit: not sure it makes much sense to have a hook for something that can be done with a regular function. It is safe to do assign value to `document.title` multiple times so no real need in `useEffect`. I'd rather create a `utils/dom.ts` file and have export a simple function called `setPageTitle` from there. But I don't feel strong about it so up to you here.